### ISSUE TITLE
[fvt]2.12.1 xcatprobe image  -c need modify the output images if compute node have the same osimage

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1415]
entv:rhels 7.2
build:latest
lsdef - Version 2.12.1 (git commit 5431a2ab9ccd42f5f76540ec0868429dd6199056, built Sun Jun 26 06:47:12 EDT 2016)
```
root@c910f02c01p12 bin]# lsdef
c910f02c01p17  (node)
c910f02c09p23  (node)
c910f02fsp01  (node)
c910f02fsp09  (node)
c910f03c05k02  (node)
c910hmc01  (node)
testnode  (node)
testnodell  (node)
[root@c910f02c01p12 bin]# xcatprobe  image -c c910f03c05k02
       c910f03c05k02 is diskless
[ OK ] Pinging c910f03c05k02
       ---- Gathering information from node c910f03c05k02 ----
[ OK ] All compute nodes have the same os image installed: rhels7.2-ppc64le-netboot-compute:fd7eeb7e-f243-4d4c-9884-d3409c8f8668.
```
I have many nodes manged by the mangement node ,but only one node have the same os image installed with xcatdb,but the output shows all comoute node ...

**Comment by gurevichmark** [comment]
Fixed by pull request #1518

**Comment by junxiawang** [comment]
verified
